ext/mysqli/mysqli.c

```diff
@@ -729,15 +653,7 @@ PHP_MINIT_FUNCTION(mysqli)
 	REGISTER_LONG_CONSTANT("MYSQLI_REPORT_OFF", 0, CONST_CS | CONST_PERSISTENT);
 
 	/* We use non-nested macros with expansion, as VC has problems */
-#ifdef MYSQLI_USE_MYSQLND
 	REGISTER_LONG_CONSTANT("MYSQLI_DEBUG_TRACE_ENABLED", MYSQLND_DBG_ENABLED, CONST_CS | CONST_PERSISTENT);
-#else
-#ifdef DBUG_ON
-	REGISTER_LONG_CONSTANT("MYSQLI_DEBUG_TRACE_ENABLED", 1, CONST_CS | CONST_PERSISTENT);
-#else
-	REGISTER_LONG_CONSTANT("MYSQLI_DEBUG_TRACE_ENABLED", 0, CONST_CS | CONST_PERSISTENT);
-#endif
```

Comment:
This seems to be unrelated to libmysql

Reply:
This is a nested `ifdef`. As I understand it would only be applicable if the whole else block would be used. The else block is not used with mysqlnd.

Reply:
Right.